frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -168,6 +168,13 @@ export class ArchivedItemDetailQA extends TailwindElement {
   }
 
   render() {
+    const fileCount = this.crawl?.filePageCount || 0;
+    const errorCount = this.crawl?.errorPageCount || 0;
```

Comment:
This is minor and not _actually_ relevant here, but generally I think `??` is preferable to `||`, if we were falling back to something other than 0 we might not want to fall back if the counts are `0`, which counts as falsy. Again, here `0` would just fall back to `0` though, so it's fine!